students/mgummel/lesson06/test_mailroom.py

```diff
@@ -101,4 +108,65 @@ def test_build_template4():
+    assert build_template(("Eddie Vedder", test_db2["Eddie Vedder"])) == email_template
+
+
+def test_file_creation1():
+   cwd_path = os.path.abspath(".")
+   write_files(cwd_path, db=test_db2)
+   assert os.path.exists(f'{cwd_path}/Gerard_Way.txt')
+   assert os.path.exists(f'{cwd_path}/Eddie_Vedder.txt')
+
+def test_file_creation2():
+   cwd_path = os.path.abspath(".")
+   write_files(cwd_path, db=test_db1)
+   assert os.path.exists(f'{cwd_path}/Paul_Allen.txt')
+   assert os.path.exists(f'{cwd_path}/Eric_Johnson.txt')
+
+
+def test_file_content1():
+    with open('./Paul_Allen.txt', 'r', encoding='utf-8') as donor_email_file:
+        email_file = donor_email_file.read()
+    email_template = '\n'.join(('\n\nDear Paul Allen,\n',
+                                'Your past donation amount of $707.10\n',
+                                'has helped our organization tremendously.\n',
+                                'Thank you for your very kind donation of $1.32.\n',
+                                'It will be put to very good use.\n',
+                                '           Sincerely,',
+                                '             -The Team\n'))
+    assert email_file == email_template
+
+
+def test_file_content2():
+    with open('./Jeff_Bezos.txt', 'r', encoding='utf-8') as donor_email_file:
+        email_file = donor_email_file.read()
+    email_template = '\n'.join(('\n\nDear Jeff Bezos,\n',
+                                'Thank you for your very kind donation of $877.33.\n',
+                                'It will be put to very good use.\n',
+                                '           Sincerely,',
+                                '             -The Team\n'))
+    assert email_file == email_template
+
+
+def test_file_content1():
+    with open('./William_Gates_III.txt', 'r', encoding='utf-8') as donor_email_file:
+        email_file = donor_email_file.read()
+    email_template = '\n'.join(('\n\nDear William Gates, III,\n',
+                                'Your past donation amount of $653772.32\n',
+                                'has helped our organization tremendously.\n',
+                                'Thank you for your very kind donation of $12.17.\n',
+                                'It will be put to very good use.\n',
+                                '           Sincerely,',
+                                '             -The Team\n'))
+    assert email_file == email_template
+
+
+def test_file_content3():
+    with open('./Gerard_Way.txt', 'r', encoding='utf-8') as donor_email_file:
+        email_file = donor_email_file.read()
+    email_template = '\n'.join(('\n\nDear Gerard Way,\n',
+                                'Thank you for your very kind donation of $76.45.\n',
+                                'It will be put to very good use.\n',
+                                '           Sincerely,',
+                                '             -The Team\n'))
+    assert email_file == email_template
```

Comment:
each one of your tests should execute the code you're testing. 

neither of `test_file_content*` functions are valid tests

Reply:
@natasha-aleksandrova I'm in a rush to complete all the exercises for this course. Can I schedule some time with you so that I can understand how I should be better writing these tests? Thank you :)

Reply:
@mgummel25 sure, send us an email and we can schedule some time to go over unit testing.

Note that I believe you should have received full grade here for this assignment.